src/resources/Incorporations/cooperative.ts

```diff
@@ -35,9 +35,7 @@ export const CooperativeResource: IncorporationResourceIF = {
           person to locate the Director.`
       ]
     },
-    addIncorporator: false,
-    addOrganization: false,
-    addProprietor: false,
+    addOrganization: 'Add Organization',
```

Comment:
It's now a string so that Cooperatives can have a different label from the other entity types. This removes the "isTypeCoop" logic from the PeopleAndRoles component.

Empty string is false, of course.